### ISSUE TITLE
Privacy: Run self-hosted tracker without cookie

### ISSUE THREAD

**Issue by gerricom** [ksensehq/eventnative#38]
First: This looks like a very promising project! Thank you very much for your work! 
Once the Clickhouse integration is finished, this could replace our own hacky solution for tracking events.

But there is one caveat and this is the GDPR and the cookie regulations we see here in Europe. Using cookies for tracking purposes requires us to inform the user and collect an opt-in (which they most likely won't do anyways... ).

Could you imagine to add a cookie-less mode for the tracking script? Some projects, like Fathom, Ackee or Goatcounter are implementing ways to track without a cookie (you can find a very interesting insight here: https://github.com/zgoat/goatcounter/blob/master/docs/sessions.markdown).

**Comment by vklimontovich** [comment]
Thank you! Cookie-less seems like a great idea, however, we'll make it optional. 

P. S. ClickHouse will be ready sometimes next week

**Comment by miwelc** [comment]
Hi! Any update on this? We are very interested in using jitsu but this issue is a stopper for us.

**Comment by vklimontovich** [comment]
Hi @miwelc. We'd very like want to implement this feature. However, I'm not sure how exactly it should work. There's number of options. Would you mind jump on a quick call or join Slack and DM me? I have some questions about your use-case.
 - [Join our Slack](http://jitsu.com/slack)
 - [Schedule a call](https://calendly.com/vklmn/jitsu-15m-call)

**Comment by xtreding** [comment]
Hi @gerricom, @miwelc! I'm happy to inform you that this feature has been released in the `1.34.3` version! 
Now you can initialize Jitsu in the strict privacy mode so cookies and client IP addresses won't be used.
- instead of cookies Jitsu will use fingerprints based on customers IP addresses and User Agents
- last IP address octet will be replaced with `1` like `10.10.10.10` -> `10.10.10.1`

Also, Jitsu supports [geo-targeting](https://jitsu.com/docs/sending-data/privacy-mode#geo-targeting)! 

[**Read more about Privacy Mode**](https://jitsu.com/docs/sending-data/privacy-mode) »
